docs/_sass/theme/variables.scss

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Asap:wght@700&display=swap');
+
+// Global variables
+$global-primary-background:                     #fc9c62;
+//$global-secondary-background:                   #433E3F;
```

Comment:
Do we need this? Can we remove it instead of commenting it out?

Reply:
It was a left over comment, thanks for spotting it!